.idea/codeStyles/Project.xml

```diff
@@ -37,15 +37,6 @@
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value />
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```

Comment:
can you revert this change?

Reply:
@macgills you mean to revert this commit  https://github.com/kiwix/kiwix-android/pull/2443/commits/8126ee70164dae5916e8823759e2bdb5fb3e36fd

I actually took the code from here https://github.com/kiwix/kiwix-android/blob/develop/.idea/codeStyles/Project.xml

Reply:
I don't want to see any changes to this file. This file is automatically generated from our settings and shared among all kiwix developers so we have consistent environments. This particular block is contributed by the kotlin plugin I believe so you may have it disabled or have an older version

Reply:
I guess the point I was trying to get across is that manually editing this file is a fools errand as it just gets rewritten by your settings. The easiest thing is just not to add any changes to this file to a commit unless you intentionally set out to change everybody's settings

Reply:
ok, I will see to it and commit the changes and make a new pull request.

Reply:
@macgills I have made the changes please review it.

Reply:
you can click the refresh symbol
![image](https://user-images.githubusercontent.com/3358282/96118485-79471a00-0ee3-11eb-8b29-abff38b5a6f5.png)
to request additional reviews and I'll get a notification

Reply:
unfortunately I am out of time today but I will most definitely review tomorrow

Reply:
Sure, thank you.